### ISSUE TITLE
Alfred MDN search stop working

### ISSUE THREAD

**Issue by TheRainManX** [gilbarbara/alfred-workflows#8]
MDN search workflow suddenly stop working today, don't know if it has anything to do with recent updates of Alfred. Can anyone help me with it? This workflow has been very helpful to me as a newbie frontend developer. Thanks   @gilbarbara @marcuslyons 
![Xnip2021-11-18_23-40-39](https://user-images.githubusercontent.com/50209936/142447362-0a69e92c-e83b-43a5-902c-e4b48b9b17a2.jpg)

**Comment by TheRainManX** [comment]
My guess is it because the API change of MDN search?

**Comment by andrew-sipes** [comment]
@lantyrs I know this is a little late, but I ran into this problem too. For me, it was not finding my `php` install. I think this has to do with macOS 12. Here is how I debugged, then fixed.

First, I opened the workflow editor and toggled the debug:

<img width="1292" alt="workflow-debug" src="https://user-images.githubusercontent.com/25012869/153735858-8128bd9b-e9d9-4c54-a142-e695310fe8b8.png">

Then, I confirmed the path to my `php` in Terminal.

```bash
which php
/opt/homebrew/bin/php
```

Finally, I added that path to the workflow, to make sure it knew where to look for `php`.

<img width="1292" alt="modify-workflow-script" src="https://user-images.githubusercontent.com/25012869/153735860-693c5460-1231-48b4-aee0-1eaea62f2e28.png">

Now it works!

<img width="623" alt="Screen Shot 2022-02-12 at 9 38 00 PM" src="https://user-images.githubusercontent.com/25012869/153735881-020a06a1-f51d-40f6-aad6-69e84cb4078a.png">

In case you didn't find a fix, I hope this is helpful. I love this workflow, and would have been so sad if I had to go without it!
